tests/process_manager/views/test_partial_views.py

```diff
@@ -26,16 +27,31 @@ def test_get(self, auth_client, mocker):
         for row, uuid in zip(table.data.data, uuids):
             assert row["uuid"] == uuid
 
-    def _mock_session_info(self, mocker, uuids):
+    def _mock_session_info(self, mocker, uuids, sessions: list[str] | None = None):
```

Comment:
I think the below works and is slightly simpler.
```suggestion
    def _mock_session_info(self, mocker, uuids, sessions: list[str] = []):
```